spec.html

```diff
@@ -1024,6 +1024,7 @@ <h1>Mathematical Operations</h1>
       <emu-note>
         <p><emu-eqn>floor(_x_) = _x_ - (_x_ modulo 1)</emu-eqn>.</p>
       </emu-note>
+      <p>An <dfn id="interval">interval</dfn> from lower bound _a_ to upper bound _b_ is a possibly-infinite, possibly-empty set of numeric values of the same numeric type. A numeric value _x_ is included in the interval if it satisfies both its <dfn>lower bound criteria</dfn> and its <dfn>upper bound criteria</dfn>. If an interval's lower bound is described as inclusive, its lower bound criteria is _a_ &le; _x_; otherwise it is _a_ &lt; _x_. If an interval's upper bound is described as inclusive, its upper bound criteria is _x_ &le; _b_; otherwise it is _x_ &lt; _b_. An <dfn id="inclusive-interval">inclusive interval</dfn> is an interval where both bounds are inclusive. It is an editorial error if a lower bound or upper bound is not described as either inclusive or exclusive. For the purpose of defining intervals, *-0*<sub>𝔽</sub> &lt; <!-- this comment here to avoid ecmarkup complaining about comparison to 0 --> *+0*<sub>𝔽</sub>. *NaN* is never included in an interval.</p>
```

Comment:
Re: wording, I find this harder to follow than my original suggestion (mostly because it's much more verbose). I won't die on this hill, but I would prefer my shorter, IMO clearer suggestion.

If we do keep this wording, it would benefit from an example or two - e.g. "For example, 'the real interval from 1 (inclusive) to 5 (exclusive)' consists of all real numbers between 1 and 5, including 1 and not including 5."

---

I would `em`, not `dfn`, the "criteria" terms. They're not used outside this paragraph.

Also the singular of "criteria" is "criterion".

Reply:
Added two examples and changed the wording for you. I hate singular "criterion", so I changed it to "condition".

Reply:
This works for me.